assets/panel/debugger.properties

```diff
@@ -92,9 +92,13 @@ ignoreExceptionsItem=Ignore exceptions
 # item shown when a user is adding a new breakpoint.
 pauseOnUncaughtExceptionsItem=Pause on uncaught exceptions
 
-# LOCALIZATION NOTE (pauseOnExceptionsItem): The pause on exceptions button description
+# LOCALIZATION NOTE (pauseOnExceptions): The pause on exceptions checkbox description
 # when the debugger will pause on all exceptions.
-pauseOnExceptionsItem=Pause on all exceptions
+pauseOnExceptions=Pause on exceptions
```

Comment:
Unfortunately this file is quite large, so you picked a string ID that already exist, and has a completely different text. That's another constraint to keep in mind, worst case scenario you can add a 2 to the existing ID (pauseOnExceptionsItem2)